_includes/_components/url-generation.html

```diff
@@ -18,11 +18,26 @@
 {% highlight php %}
 <?php
 
-class Test
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+class TestController
 {
-    public function test(SomeClass $subject)
+    /**
+     * @Route("/s/{shopId}", name="shop_show")
+     */
+    public function showAction($shopId)
+    {
+        return $this->redirectToRoute('shop_products', [
+            'shopId' => $shopId,
+        ]);
+    }
+
+    /**
+     * @Route("/s/{shopId}/p", name="shop_products")
```

Comment:
In fact, it's too long, if I write all the word, then the comment will be splitted into 2 lines...